tests/E2E/ErrorTest.php

```diff
@@ -40,20 +40,23 @@ public function testAccessTokenInvalid()
      */
     public function testInvalidQuery()
     {
-        $client = new Client('b4c0n73n7fu1', 'cfexampleapi');
+        $client = $this->getClient('cfexampleapi');
+
         $query = (new Query())
             ->where('name', 0);
 
         $client->getEntries($query);
     }
 
     /**
+     * @requires Contentful
```

Comment:
While I'm not the biggest annotations fan, this seems fine.

What I don't think is good is the name. All E2E tests require Contentful, what makes these different?